src/seismometer/__init__.py

```diff
@@ -72,3 +72,57 @@ def run_startup(
     # Surface api into namespace
     s_module = importlib.import_module("seismometer._api")
     globals().update(vars(s_module))
+
+
+def download_example_dataset(dataset_name: str, branch_name: str = "main"):  # pragma: no cover
+    """
+    Downloads an example dataset from the specified branch to local data/ directory.
```

Comment:
Sphinx really likes blank lines (and I dont see the option for multiline suggestions?)

Can you add a blank line before Parameters and before Raises,  this will fix the sphinx generation

Reply:
FINC